CHANGES.md

```diff
@@ -517,7 +517,7 @@ OpenSSL 3.2
    *Paul Dale*
 
  * Subject or issuer names in X.509 objects are now displayed as UTF-8 strings
-   by default.
+   by default. Also spaces surrounding `=` in DNs output are removed.
```

Comment:
(minor) I don't think DN needs to be plural here: it's being used more like an adjective than a noun, since there isn't a verb in "DN output". It would be plural in "output of DNs". i.e. this reads better as

```
Also spaces surrounding `=` in DN output are removed.
```